pkg/arm/data/etc/origin/node/pods/apiserver.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ''
+  labels:
+    openshift.io/component: api
+    openshift.io/control-plane: 'true'
+  name: master-api
+  namespace: kube-system
+spec:
+  containers:
+  - args:
+    - start
+    - master
+    - api
+    - --config=/etc/origin/master/master-config.yaml
+{{- if $.Extra.TestConfig.RunningUnderTest }}
```

Comment:
@charlesakalugwu note #1175 is touching the same files as this PR

Reply:
Roger